pyanaconda/errors.py

```diff
@@ -106,8 +106,6 @@ def _get_default_mapping(self):
             PayloadInstallError.__name__: self._payload_install_handler,
 
             # DNF errors
-            "NoStreamSpecifiedException": self._no_module_stream_specified,
-            "InstallMoreStreamsException": self._multiple_module_streams_specified,
```

Comment:
Hmm, could have been something that was planned but never implemented/handled in a different manner for modularity in the end. Should be safe to drop.